DevTools/ContainerDebug/NanoDockerBuild.cmd

```diff
@@ -50,17 +53,25 @@ if %ERRORLEVEL% GTR 8 (
     exit /b 1
 )
 
-call robocopy.exe %VS_CRT_REDIST_DBG% %LOCAL_BIN_STAGING_DIR% /S /E >nul
-if %ERRORLEVEL% GTR 8 (
-    echo Error while staging debugger binaries. Exiting...
-    exit /b 1
+for /L %%n in (0,1,1) do (
+    call robocopy.exe !VS_CRT_REDIST_REL[%%n]! %LOCAL_BIN_STAGING_DIR% /S /E >nul
```

Comment:
One last comment, I believe this should be `VS_CRT_REDIST_DBG`. Other than that, looks good.

Reply:
fixed